bats/foreign-keys.bats

```diff
@@ -1404,3 +1404,383 @@ SQL
     [[ "$output" =~ "4,4,4" ]] || false
     [[ "${#lines[@]}" = "3" ]] || false
 }
+
+@test "foreign-keys: Commit all" {

```

Comment:
New tests are great, but you don't have any that test validating the foreign key constraints after a merge.